src/MSBuild.Bootstrap/MSBuild.Bootstrap.csproj

```diff
@@ -36,7 +36,7 @@
 
     <Content Include="$(MSBuildExtensionsPath)\**\*" LinkBase="Extensions" CopyToOutputDirectory="PreserveNewest" />
 
-    <Content Include="$(RepoRoot).dotnet\sdk\$(DotNetCliVersion)\RuntimeIdentifierGraph.json" CopyToOutputDirectory="PreserveNewest" />
+    <Content Include="$(RuntimeIdentifierGraphPath)" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
I think this was a similar situation with `NETCoreSdkBundledVersionsProps`, I'll try reverting it